src/ontology/mondo-edit.obo

```diff
@@ -465430,7 +465434,7 @@ is_a: MONDO:0015977 {source="MESH:C538056", source="MONDO:Redundant", source="OM
 intersection_of: MONDO:0011096 ! autosomal agammaglobulinemia
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/5541 ! IGHM
 relationship: curated_content_resource https://search.clinicalgenome.org/kb/conditions/MONDO:0020729 {source="MONDO:CLINGEN"}
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/5541 {source="MONDO:mim2gene_medgen", source="OMIM:601495"} ! IGHM
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/5541 {source="MONDO:mim2gene_medgen"} ! IGHM
```

Comment:
It seems this OMIM source should have been maintained. @joeflack4 can you look into this?

Reply:
So, the phenotype OMIM:601495 is mapped to the gene OMIM:147020 as shown in `morbidmap.txt`:

`Agammaglobulinemia 1, 601495 (3)	IGHM, MU, AGM1	147020	14q32.33`

But as with my [long explanation above](https://github.com/monarch-initiative/mondo/pull/8624#discussion_r1933067509), the reason we don't have any HGNC source evidence being added is because it only shows an NCBI gene entry in `mim2gene.txt`, not an HGNC one:

`mim2gene.txt`:
`147020	gene	3507		ENSG00000211899`

Which gets added like this in `omim.owl`:
`AnnotationAssertion(<http://www.w3.org/2004/02/skos/core#exactMatch> <https://omim.org/entry/147020> <https://www.ncbi.nlm.nih.gov/gene/3507>)

Reply:
The issue w/ the data that Trish and I are discussing is that we see a label for the HGNC term in `morbidmap.txt`, but the ID for that term does not appear in `mim2gene.txt`. We will contact OMIM about this apparent discrepancy.

Reply:
The OMIM source provenance was added back to this gene association.